drivers/serial/uart_stellaris.c

```diff
@@ -644,7 +644,7 @@ static struct uart_stellaris_dev_data_t uart_stellaris_dev_data_0 = {
 	.baud_rate = TI_STELLARIS_UART_4000C000_CURRENT_SPEED,
 };
 
-DEVICE_AND_API_INIT(uart_stellaris0, CONFIG_UART_STELLARIS_PORT_0_NAME, &uart_stellaris_init,
+DEVICE_AND_API_INIT(uart_stellaris0, "UART_0", &uart_stellaris_init,
```

Comment:
Why this change, we should keep things as they are and use label in device tree like other uarts are using.